guides/doc-Installing_Satellite_Server_Disconnected/master.adoc

```diff
@@ -132,18 +132,18 @@ include::common/modules/configuring_satellite_external_tftp.adoc[leveloffset=+2]
 // Configuring the Firewall for External TFTP Service
 include::common/modules/configuring_satellite_external_tftp_firewall.adoc[leveloffset=+3]
 
-// Configuring Satellite with External IdM DNS
+// Configuring {Project} with External IdM DNS
 include::common/modules/configuring_satellite_external_idm_dns.adoc[leveloffset=+2]
 
-== Uninstalling Satellite Server
+== Uninstalling {ProjectServer}
 
-// Uninstalling Satellite Server and Capsule Server
+// Uninstalling {ProjectServer} and Capsule Server
 include::common/modules/uninstalling_parent.adoc[leveloffset=+1]
 
-// Uninstalling Satellite Server
+// Uninstalling {ProjectServer}
 include::common/modules/uninstalling_satellite_server.adoc[leveloffset=+2]
 
 :numbered!:
 
-// Applying Custom Configuration to Red Hat Satellite
-include::common/modules/applying_custom_configuration_to_red_hat_satellite.adoc[leveloffset=+1]
+// Applying Custom Configuration to {Project}
+include::common/modules/applying_custom_configuration_to_red_hat_{Project}.adoc[leveloffset=+1]
```

Comment:
I don't know why it did this only for the Disconnected master and not Connected, but it breaks the include. It renders at the end of the book as `Unresolved directive in master.adoc - include::common/modules/applying_custom_configuration_to_red_hat_Satellite.adoc[leveloffset=+1]`

Reply:
It does this randomly. It's very annoying.